app/views/hq/users/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_for [:hq, @user] do |f| %>
+  <%= render 'shared/errors', object: f.object %>
+  <div class="form-group">
+    <%= f.label :user_name %>
+    <%= f.text_field :user_name, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= f.label :email %>
+    <%= f.text_field :email, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= f.submit class: "btn btn-default" %>
+    <%= link_to 'Cancel', hq_user_path(@user), class: "btn" %>
```

Comment:
This will probably raise an exception if the form is used for `new` as `@user.id` will be `nil`.  Suggest a conditional link destination